.travis.yml

```diff
@@ -77,6 +72,10 @@ jobs:
         - cmake --build ringmesh --target package
         - cd $TRAVIS_BUILD_DIR
         - sonar-scanner
+      after_failure:
+        - if [ "$TRAVIS_BRANCH" = "master" ]; then 
+          chmod 777 ./../travis_slack_notif.sh;
+          ./../travis_slack_notif.sh "$TRAVIS_JOB_ID" "$TRAVIS_BRANCH"; fi
```

Comment:
You could probably remove your chmod and execute using "bash $TRAVIS_BUILD_DIR/travis_slack_notif.sh ..."